### PR TITLE
Changed imageupload to fileupload and added link on template view

### PR DESCRIPTION
**Pull request by nakul2234**

### CODE REVIEW

**Comment by nakul2234** [comment]
@justinrporter I made a small change to the code that should allow us to support both images and pdf documents. Since imagefield is a type of filefield, it was easy to migrate the database for imagefield into filefield. Now, pdfs can be uploaded and the Document detail page will have a broken picture img and a link to the file below.

**Comment by justinrporter** [comment]
addresses #32

**Comment by nakul2234** [comment]
As we discussed, just having the link will work for now. Should be good to go so we can start using pdf files.

File: pttrack/models.py [hunk 1]
```diff
@@ -371,9 +371,10 @@ def __unicode__(self):
 
 class Document(Note):
     title = models.CharField(max_length=200)
-    image = models.ImageField(
+    image = models.FileField(
         help_text="Please deidentify all file names before upload!",
```
Comment (justinrporter) [inline pttrack/models.py]: we can probably ditch this `help_text` now.

File: pttrack/templates/pttrack/document_detail.html [hunk 2]
```diff
@@ -28,10 +28,15 @@
...
     <br><small class="text-muted">last modified {{document.last_modified}}</small>
-    {% endif %}</p>
+    {% endif %}
+  <div class="row">
+    <h3><br>&nbsp;&nbsp;<a href="{{ document.image.url }}">Link to File</a></h3>
+    <small class="text-muted">&nbsp;&nbsp;&nbsp;&nbsp;(Preview and Comments Below)</small>
+  </div>
   <div class="row ">
-    <img class="img-rounded img-responsive center-block" src="{{ document.image.url }}" alt="alt_text">
+    <img class="img-rounded img-responsive center-block" src="{{ document.image.url }}" alt="Please Click Link Below">
```
Comment (justinrporter) [inline pttrack/templates/pttrack/document_detail.html]: technically speaking, we should be using the alt-text to describe the file, and not to give instructions (it's an accessibility feature for blind people, I think). That being said, I'm not sure how important that is.

Comment (nakul2234) [inline pttrack/templates/pttrack/document_detail.html]: Yeah that is temporary I think. Once we move to PDF I do not think we should preview it, there will just be a link people can click and then see the file. Embedding the pdf seems like a bad idea for ones with multiple pages and such.

File: pttrack/models.py (hunk 1 above)
Comment (nakul2234) [inline pttrack/models.py]: I think that is an important reminder when handling these documents. That way the file on their harddrive will also be deidentified and they remmber to do it every time.

Comment (justinrporter) [inline pttrack/models.py]: Perhaps a better reminder would be to delete the file after uploading, then?

File: pttrack/templates/pttrack/document_detail.html (hunk 2 above)
Comment (justinrporter) [inline pttrack/templates/pttrack/document_detail.html]: Hm. I see your point, but I am also not sure I want to be encouraging people to download phi to their hard drives, lest they forget to delete it when they're finished...

Comment (nakul2234) [inline pttrack/templates/pttrack/document_detail.html]: Most browsers will jsut open the file right? So they would have to physically save the file for it to work? I can look into embedding or pdf viewer software.

File: pttrack/models.py (hunk 1 above)
Comment (nakul2234) [inline pttrack/models.py]: Makes sense. Ill make a quick edit.

File: pttrack/templates/pttrack/document_detail.html (hunk 2 above)
Comment (justinrporter) [inline pttrack/templates/pttrack/document_detail.html]: Oh! You may be right--I am honestly not sure what the default behavior is, and it's likely dependent. If Chrome and Firefox both do it, that's probably good enough for now.

Comment (nakul2234) [inline pttrack/templates/pttrack/document_detail.html]: Wow, I overestimated how complicated embedding a pdf would be. It is fully functional and looks pretty damn nice!

File: pttrack/models.py (hunk 1 above)
Comment (nakul2234) [inline pttrack/models.py]: Added a line reminding to delete the files too

**Comment by nakul2234** [comment]
Also, git is showing all these changes to xml files which I am not sure if I understand? Maybe it has something to do with having pdf files and the pdf viewer of the browser displayed?

**Pull request merged**